pkg/util/util.go

```diff
@@ -51,6 +54,8 @@ const (
 	// Users can use RolloutIDLabel and RolloutBatchIDLabel to select the pods that are upgraded in some certain batch and release.
 	RolloutBatchIDLabel = "apps.kruise.io/rollout-batch-id"
 	WorkloadTypeLabel   = "rollouts.kruise.io/workload-type"
+	// RollbackInBatchAnnotation allow use disable quick rollback, and will roll back in batch style.
+	RollbackInBatchAnnotation = "rollouts.kruise.io/rollback-in-batch"
```

Comment:
can we use rollback-in-batch by default if the workload supported, we can simulate rollback directly by setting only a single steps